src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -89,10 +89,21 @@
       <Link>Common\System\HResults.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' and ('$(TargetGroup)' == 'uapaot' or '$(TargetGroup)' == 'uap')">
+    <Compile Include="System\Environment.WinRT.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' and '$(TargetGroup)' != 'uapaot' and '$(TargetGroup)' != 'uap'">
```

Comment:
We usually try to be inclusive in the conditions as opposed to exclusive. Can you try something like `TargetGroup == netcoreapp`